src/main/resources/view/PersonListCard.fxml

```diff
@@ -9,6 +9,7 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
 
+<?import com.sun.javafx.charts.Legend?>
```

Comment:
just wondering, is `Legend` being used in `PersonListCard.fxml`? Don't seem to see it here

Reply:
Careless mistake. Resolved!